README.md

```diff
@@ -69,3 +69,13 @@ docker run -it --rm -p 8888:8888 pangeo/base-notebook:latest jupyter lab --ip 0.
 Everything stems from the `Dockerfile` in the `base-image` folder. The `base-image` configures default settings for Conda and Dask with `condarc.yml` and `dask_config.yml` files. The `base-image` is not meant to run on its own, it is the common foundation for `-notebook` images that install Python packages including JupyerLab and lab extensions. Lists of Conda packages for each image are specified in an `environment.yml` in each `-notebook` folder, and compatible Dask and Jupyter packages are guaranteed by specifying the `pangeo-notebook` [conda metapackage](https://github.com/conda-forge/pangeo-notebook-feedstock).
 
 You can pre-solve for compatible environments locally with [conda-lock](https://github.com/mariusvniekerk/conda-lock/blob/master/README.md) to convert the `environment.yml` file to a [conda-linux-64.lock](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#building-identical-conda-environments) file which is an explicit list of compatible packages solved by Conda. The major advantage of doing this is that if you rebuild at a later date the resulting Conda environment is identical, which improves reproducibility. For this reason, when building off of the `base-image`, any existing `conda-linux-64.lock` file takes precedence over the `environment.yml` file.
+
+### Environment
+
+The runtime environment sets two variables by default
+
+1. `$PANGEO_ENV`: name of the conda environemnt.
+2. `$PANGEO_SCRATCH`: a URL like `gcs://pangeo-scratch/username/` that
+   points to a cloud storage bucket for temporary storage. This is set
+   if the variable `$PANGEO_SCRATCH_PROTOCOL` and `JUPYTERHUB_USERNAME`
+   are detected.
```

Comment:
This seems inconsistent with the start script. The script only checks for PANGEO_SCRATCH_PROTOCOL. If JUPYTERHUB_USERNAME is empty, PANGEO_SCRATCH will get set to `${PANGEO_SCRATCH_PROTOCOL}://pangeo-scratch/`. That could be different from what we want.